utils.sh

```diff
@@ -68,6 +68,22 @@ function create_cluster() {
     $OPENSHIFT_INSTALLER --dir "${assets_dir}" --log-level=debug wait-for install-complete
 }
 
+function ipversion(){
+    if [[ $1 =~ : ]] ; then
+        echo 6
+        exit
+    fi
+    echo 4
+}
+
+function wrapifipv6(){
```

Comment:
Nit, could we make this snake case (wrap_if_ipv6) to conform to our other function names?

Reply:
yup